README.rst

```diff
@@ -1,21 +1,95 @@
+Repository for building and executing the `OCTOPUS code
+<http://octopus-code.org>`__ package in a Docker container. 
+
+Use case: run Octopus (for small calculations and tutorials) conveniently in
+container, in particular on MacOS and Windows where compilation of Octopus may be non trivial.
+
+
 Octopus in Docker container
 ===========================
 
-.. sectnum::
+Quick start
+-----------
 
-.. contents:: 
 
-Support repository for building and executing the `OCTOPUS code <http://octopus-code.org>`__ package in a Docker container. (Other container formatst [singularity?] could be created from this.)
+1. `Install docker <https://docs.docker.com/get-docker/>`__ on your machine.
 
-Use cases: run octopus (for small calculations and tutorials) conveniently in container, in particular on MacOS and Windows.
+   Check to confirm: run ``docker --version``. Expected output is something like this::
 
+     $ docker --version
+     Docker version 20.10.12, build e91ed57
 
-Status 
-------
+2. Change into the directory that contains your ``inp`` file.
 
-|stable| Debian Bookworm (12), Last octopus release (13.0)
 
-|develop| Debian Bookworm (12), Octopus develop branch
+3. Then run::
+
+    docker run --rm -ti -v $PWD:/io fangohr/octopus octopus
+  
+   The first time you run this, Docker needs to download the image
+   ``fangohr/octopus`` from DockerHub. This could take a while (depending on your
+   internet connection, the image size is about 850MB).
+
+   Meaning of the switches:
+   
+   - ``--rm`` remove docker container after it has been carried out (good practice to reduce disk usage).
+   - ``-ti`` allow to make the Terminal session Interactive
```

Comment:
perhaps explain the use of both ``t`` and ``i`` flags (see https://docs.docker.com/engine/reference/run/#commands-and-arguments)
```suggestion
   - ``-ti`` start an ``i``nteractive  pseudo-``t``ty shell in the container 
```